Update Minikube FusionAuth tutorial (#1409)
FusionAuth/fusionauth-site: site/docs/v1/tech/installation-guide/kubernetes/setup/minikube.adoc

[source,shell,title=Install the postgresql chart]
----
helm install pg-minikube --set auth.postgresPassword=<your-postgres-password> bitnami/postgresql
----

> Setting the password is optional and will be generated automatically if you do not set one.

Reviewer: ```suggestion
Setting the password is optional. A password will be generated automatically if you do not set one.
```

    PGPASSWORD="$POSTGRES_PASSWORD" psql --host 127.0.0.1 -U postgres -d postgres -p 5432
----

> When you deploy FusionAuth, you will need to use the DNS name `pg-minikube-postgresql.default.svc.cluster.local` as seen above and the password created from installing the helm chart.

Reviewer: ```suggestion
When you deploy FusionAuth, you will need to use the DNS name `pg-minikube-postgresql.default.svc.cluster.local` as seen above and the postgres user password generated during the postgres helm chart installation.
```

[source,helmtext,title=Export the Postgres password]
----
export POSTGRES_PASSWORD=$(kubectl get secret --namespace default pg-minikube-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
----

> Following the above command, if you did not set a password when installing my-release you can use the command below to display it.

Reviewer: ```suggestion
If you did not set a password when installing `my-release`, use the command below to display the value of the `POSTGRES_PASSWORD` environment variable.
```

[source,helmtext, title=Display your password]
----
echo $POSTGRES_PASSWORD
----

> You can also test your deployment by attempting to connect to the database following the instructions in the output.

Reviewer: ```suggestion
You can also test your deployment by attempting to connect to the database.
```

I don't know what 'instructions in the output' means? Are you saying that this command outputs instructions you then need to copy and paste?
